packages/web-app/src/views/review-complete.njk

```diff
@@ -0,0 +1,25 @@
+{% extends "layouts/form.njk" %}
+
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% block backButton %}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-!-margin-bottom-7" >
+        {{ govukPanel({
+          titleText: pageTitle,
+          html: checkAndConfirmConfig.reviewCompleteStatusText + "<br><strong>" + appealData.horizonId + "</strong>"
+        }) }}
+      </div>
+
+      <p class="govuk-body">{{ checkAndConfirmConfig.reviewCompleteText }}</p>
+
+      <h2 class="govuk-heading-m">What next?</h2>
+
+      <p class="govuk-body"><a href="/appeals-list" class="govuk-link">Return to task list</a></p>
+    </div>
+  </div>
+{% endblock %}
```

Comment:
This page isn't a form so I don't think that it needs to extend the form layout, probably extending the two thirds page layout would be better otherwise the page will inherit other form-specific tags that come with the form layout.

However as the `content` block is being used here both the form and the two thirds page layouts are being skipped as the `content` block is defined in the main layout which is at the top level and this is why the grid layout tags have needed to be added. If the correct block is used then these will be inherited from the two thirds page layout. 

For the form layout the block to use is `formContent` and for the two thirds page layout the block to use is `pageContent`.